neuror/main.py

```diff
@@ -464,9 +488,9 @@ def _get_order_offset(self, branch):
         the branch order of the first oblique section so we have to remove the offset.
 
                     3
-                 2  |
-     oblique ---->\ |
-                   \| 1
+                2  |
+    oblique ---->\ |
+                    \| 1
```

Comment:
wasn't it better before?

Reply:
yes, I screw up the revert of my black...